device/bgp/routepathattributes.yaml

```diff
@@ -5,15 +5,28 @@ components:
         Configuration for advanced BGP route range settings.
       type: object
       properties:
+        include_multi_exit_discriminator:
+          description: >-
+            BGP Multi Exit Discriminator attribute sent to the peer to help in the route selection process. 
+            If set to true, the Multi Exit Discriminator attribute will be included in the route advertisement.
+          type: boolean
+          default: true
+          x-field-uid: 3
```

Comment:
what is x-field-uid?

Reply:
@SuryyaKrJana please touch base w/ Ashutosh and others to understand what this field is and why it is required.  It is very important to get this field right in the models.

Reply:
x-field-uid pins a attribute to a particular index otherwise the attribute value gets overwritten with other attribute value